package.json

```diff
@@ -8,125 +8,125 @@
     "deep-freeze": "0.0.1",
-    "detect-browser": "^1.5.0",
+    "detect-browser": "^1.7.0",
     "diacritics": "^1.3.0",
-    "draft-js": "^0.10.0",
-    "draft-js-export-html": "^0.5.2",
-    "enzyme": "^2.4.1",
-    "eslint": "^3.17.1",
+    "draft-js": "^0.10.1",
+    "draft-js-export-html": "^0.5.4",
+    "enzyme": "^2.8.2",
+    "eslint": "^3.19.0",
     "eslint-config-defaults": "9.0.0",
     "eslint-plugin-babel": "^4.1.1",
     "eslint-plugin-flow-vars": "^0.5.0",
-    "eslint-plugin-flowtype": "^2.30.3",
-    "eslint-plugin-mocha": "^4.8.0",
-    "eslint-plugin-react": "^6.7.1",
-    "express": "^4.15.2",
-    "fetch-mock": "^5.9.4",
-    "file-loader": "^0.10.1",
+    "eslint-plugin-flowtype": "^2.33.0",
+    "eslint-plugin-mocha": "^4.9.0",
+    "eslint-plugin-react": "^7.0.1",
+    "express": "^4.15.3",
+    "fetch-mock": "^5.11.0",
+    "file-loader": "^0.11.1",
     "flow-bin": "^0.43.1",
     "history": "^4.6.0",
     "iflow-lodash": "^1.1.16",
     "iflow-react-router": "^1.1.16",
     "imports-loader": "^0.7.1",
-    "iso-3166-2": "^0.5.0",
+    "iso-3166-2": "^0.6.0",
     "isomorphic-fetch": "2.2.1",
-    "jquery": "^2.2.0",
-    "jsdom": "^9.9.1",
-    "jsdom-global": "^2.1.1",
-    "keycode": "2.1.1",
+    "jquery": "^3.2.1",
+    "jsdom": "^11.0.0",
+    "jsdom-global": "^3.0.2",
+    "keycode": "^2.1.9",
     "lodash": "^4.17.4",
-    "material-ui": "^0.16.3",
+    "material-ui": "^0.18.1",
     "minimist": "^1.2.0",
-    "mocha": "^3.2.0",
-    "moment": "^2.17.0",
-    "node-sass": "^4.3.0",
+    "mocha": "^3.4.2",
+    "moment": "^2.18.1",
+    "node-sass": "^4.5.3",
     "nyc": "^10.1.2",
     "object.entries": "^1.0.4",
-    "postcss-loader": "^1.3.3",
+    "postcss-loader": "^2.0.5",
+    "prop-types": "^15.5.10",
     "ramda": "^0.23.0",
-    "raven-js": "^3.12.1",
-    "react": "^15.4.0",
+    "raven-js": "^3.15.0",
+    "react": "^15.5.4",
     "react-addons-shallow-compare": "^15.4.0",
-    "react-addons-test-utils": "^15.4.0",
-    "react-bootstrap": "^0.30.8",
-    "react-cropper": "^0.10.1",
-    "react-datepicker": "^0.41.1",
+    "react-bootstrap": "^0.31.0",
+    "react-cropper": "^0.12.0",
+    "react-datepicker": "^0.46.0",
     "react-document-title": "^2.0.3",
-    "react-dom": "^15.4.0",
-    "react-draft-wysiwyg": "1.9.3",
+    "react-dom": "^15.5.4",
+    "react-draft-wysiwyg": "^1.10.0",
     "react-dropzone": "^3.6.0",
-    "react-ga": "^2.1.2",
+    "react-ga": "^2.2.0",
     "react-geosuggest": "^2.0.0",
     "react-hot-loader": "^3.0.0-beta.6",
-    "react-mdl": "1.5.4",
-    "react-page-click": "^3.0.0",
+    "react-mdl": "^1.10.3",
+    "react-page-click": "^4.0.1",
     "react-redux": "^5.0.3",
     "react-router": "^3.0.0",
     "react-select": "^1.0.0-rc.1",
     "react-slick": "^0.14.4",
-    "react-swipeable": "^3.6.0",
+    "react-swipeable": "^4.0.0",
     "react-tap-event-plugin": "^2.0.1",
     "react-telephone-input": "^4.0.1",
+    "react-test-renderer": "^15.5.4",
```

Comment:
I believe enzyme needed it as a peer dependency, but it wasn't installed and it was throwing an error, but I can't remember the exact detail